src/Provider/Provider.Web/appsettings.json

```diff
@@ -56,5 +56,8 @@
     "TableStorage": ""
   },
   "GoogleMapsPrivateKey": "",
+  "FaaConfiguration": {
+    "StorageConnectionString": ""
+  },
```

Comment:
Have you done the corresponding ARM template change?

Reply:
Done it now.